File: Fantasy_Football/Models/Player.cs
namespace Fantasy_Football.Models
{
  

        public class PlayerModel
        {
            public int season { get; set; }
            public string format { get; set; }
            public Player[] players { get; set; }
        }

        public class Player
        {
            public string playerId { get; set; }
            public string name { get; set; }
            public string team { get; set; }
            public string position { get; set; }
            public int rank { get; set; }
            public int rank_position { get; set; }
            public string injury_risk { get; set; }
        }



    
}

File: Fantasy_Football/ClientApp/src/app/models/player.ts
export interface Player {
        season:  number;
        format:  string;
        players: PlayerElement[];
    }
    
    export interface PlayerElement {
        playerId:      string;
        name:          string;
        team:          string;
        position:      Position;
        rank:          number;
        rank_position: number;
        injury_risk:   InjuryRisk;
    }
    
    export enum InjuryRisk {
        Empty = "",
        High = "high",
        Low = "low",
        Medium = "medium",
    }
    
    export enum Position {
        Def = "DEF",
        K = "K",
        Qb = "QB",
        Rb = "RB",
        Te = "TE",
        Wr = "WR",
    }
    

